.github/workflows/csslint.yml

```diff
@@ -0,0 +1,13 @@
+name: Stylesheet Lint
+on: [push, pull_request]
+
+jobs:
```

Comment:
make tab 2 space not 4

Reply:
I changed my tabs to 2 spaces instead of 4.